great_expectations/core/expectation_validation_result.py

```diff
@@ -371,6 +371,33 @@ def get_metric(self, metric_name, **kwargs):
             )
         )
 
+    def get_failed(self):
```

Comment:
Could we perhaps rename this to `get_failed_validation_results` just to be very explicit?

Additionally, could we add `ExpectationSuiteValidationResult` as a return type? We're trying to make a more conscious effort to type hint so any additions are greatly appreciated!